### PR TITLE
update: Adding constants.

### PR DESCRIPTION
**Pull request by gkaretka**
- Adding FORWARD and BACKWARD.

Signed-off-by: Gregor Karetka gregorkaretka999@gmail.com

### CODE REVIEW

File: motor.h [hunk 1]
```diff
@@ -2,6 +2,8 @@
 #define MOTOR_H
 
 #include "Arduino.h"
```
Comment (pepol) [inline motor.h:4]: Separate includes from defines with 2 blank lines, please.

File: motor.h [hunk 2]
```diff
@@ -2,6 +2,8 @@
...
+#define FORWARD 0
+#define BACKWARD 1
```
Comment (pepol) [inline motor.h]: Wouldn't it be better to prefix those constants with `MOTOR_` for example? As to distinguish from other `FORWARD` or `BACKWARD` constants that might be used in the future.

**Comment by pepol** [comment]
@magic-sudo looks good, but please address those 2 comments.

File: motor.h (hunk 2 above)
Comment (gkaretka) [inline motor.h]: Do you mean MOTOR_FORWARD and MOTOR_BACKWARD

**Comment by pepol** [comment]
LGTM, please squash

File: motor.h [hunk 3]
```diff
@@ -1,7 +1,10 @@
 #ifndef MOTOR_H
 #define MOTOR_H
 
+
```
Comment (pepol) [inline motor.h]: I didn't mean here. I mean to separate the `#include` from the `#define` below it, not above it. That is:

``` cpp
#include "Arduino.h"


#define MOTOR_FORWARD 0
#define MOTOR_BACKWARD 1
```

**Comment by pepol** [comment]
LGTM! Please squash!

**Pull request merged**